Packages/com.chisel.editor/Chisel/Editor/UtilityWindows/SurfaceEditorWindow/SurfaceEditor.GUI.cs

```diff
@@ -0,0 +1,302 @@
+/*
+ * $TODO: clean up layout code - headers can be put into their own methods, etc.
+ */
+
+using Chisel.Components;
+using Chisel.Core;
+using UnityEditor;
+using UnityEngine;
+using EditorGUILayout = UnityEditor.EditorGUILayout;
+
+namespace Chisel.Editors
+{
+    internal partial class SurfaceEditor
+    {
+        private float           m_PanIncrementU   = 1.0f;
+        private float           m_PanIncrementV   = 1.0f;
+        private float           m_RotIncrement    = 15.0f;
+        private float           m_ScaleIncrementU = 1.0f;
+        private float           m_ScaleIncrementV = 1.0f;
```

Comment:
Eventually these will need to come from surfaces & what we really want is that we support editing multiple surfaces at the same time (as in, I've selected multiple surfaces and change a particular property once)